app/server/README.md

```diff
@@ -0,0 +1,7 @@
+# WODIN
```

Comment:
This is the README which is picked up by npm when we publish the package, as we do that from `app/server`.